assignments/prototypes.js

```diff
@@ -31,8 +56,24 @@
   * greet() // prototype method -> returns the string '<object name> offers a greeting in <object language>.'
   * should inherit destroy() from GameObject through CharacterStats
   * should inherit takeDamage() from CharacterStats
-*/
+*/  
+
+
+function Humanoid (attributes){
+  this.team = attributes.team;
+  this.weapons = attributes.weapons;
+  this.language = attributes.language;
+  GameObject.call(this,attributes)
+  CharacterStats.call(this ,attributes);
+}  
+Humanoid.prototype = Object.create(GameObject.prototype); 
```

Comment:
You shouldn't need to pull in the GameObject prototype because you're already pulling in the CharacterStats prototype which should inherit the GameObject prototype.